src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -241,16 +240,16 @@ public static synchronized void loadExternalModules(File file) { // A bit hacky,
 			} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException | IOException | ClassNotFoundException e) {
 				Discord4J.LOGGER.error(LogMarkers.MODULES, "Unable to load module " + file.getName() + "!", e);
 			}
-
 		}
 	}
 
-	private static void loadClass(String clazz) throws ClassNotFoundException{
+	private static Class loadClass(String clazz) throws ClassNotFoundException{
```

Comment:
There is no point to this method. Firstly it is only referenced once and its a short enough snippet to not warrant a method, additionally it makes literally no sense to me.